documentation/final-deliverables/pdf-report.md

```diff
@@ -2,110 +2,109 @@
+FR17. A verification email will be sent to the address provided in account registration to Mentors and Admins. The user must click that link to be able to login to their account so that they can access the features specific to their user type.
 
-FR23. To "unmatch" Mentees and Mentors, the Mentee can remove themselves from the calendar, cancelling all future appointments. The Mentor could then remove them for their dashboard on the Mentorship Portal.
+FR18. Mentors and Admins will be able to login through MentorMatch's login page. The login will be unsuccessful if either the email or password credentials do not align with what’s in the database (in which case, an error message will be displayed).
 
-FR24. Mentors who validate their email address will be able to login, however, they will not be able to view any matched Mentees until an Admin approves their account.
+FR19. Mentors and Admins will be able to reset their password by clicking on the “Forgot your password?” link on the Login page. The user will be required to enter an email address which has been associated with an account in the database. If successful, an email will be sent to that address, which the user can click to be brought to a separate page to reset their password.
 
-FR25. Admins will be able to approve Mentors through the Mentorship Portal.
+FR20. Mentors and Admins will be able to logout of their account, to which they will return to the login page, where they can login, retrieve their password, or register.
 
-FR26. Ensure all components of the system are functional, fixing any final bugs.
+FR21. Logged in Mentors will be able to accept Mentees that they have matched with (based off of similar skills, common organization name, and time-zone).
 
-FR27. Implement “bonus features” (a.k.a features that are not required of the system, but that would be nice to have):
+FR22. Once a Mentor has accepted a Mentee, an email will be sent to the Mentee to notify them of a successful match. The two can then correspond through email, and the Mentee will be able to set a time to meet with the Mentor via the Mentor's calendar link.
 
-* Allow Mentees to create an account with the system, and develop surrounding UI, with additional features.
-* Allow Mentors the ability to continue receiving Mentee requests, or put them on hold.
-* Allow a user to be both a Mentor and Mentee
-* Allow Admins to be associated with more than one organization
-* Allow the system to support more than one language.
-* Allow Admins to pass their admin privileges to another user if that current admin no longer wishes to sustain an Admin role.
+FR23. To "unmatch" Mentees and Mentors, the Mentee can remove themselves from the calendar, cancelling all future appointments.
```

Comment:
Again, I am afraid this is impossible, unless I am misunderstanding the Google Calendar documentation, or we assume that mentors add each mentee to their calendar manually.

Reply:
I debated whether or not I should keep this FR tbh. The client hasn't mentioned anything about how Mentors and Mentees should unmatch within our system since the creation of this FR back in October when we made the 'Requirements Report' milestone. Given this, I could possibly remove this FR for now, and ask the client about it during tomorrow's meeting?

Reply:
How about change it to 
```suggestion
FR23. To "unmatch" Mentees and Mentors, the Mentee can communicate with the Mentor in order to cancel all future appointments, which the Mentor can reflect on his / her calendar. 
```

I think either that or remove it, since our system just doesn't support this.

Reply:
I'll leave it for now (with the changes that you've suggested). I'll mention it to the client tomorrow during the meeting just to double check that 'unmatching' isn't an important feature (which again, it hasn't been mentioned since October, so I doubt it's important).